src/main.rs

```diff
@@ -14,23 +14,36 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <https://www.gnu.org/licenses/>. */
 
+#![feature(plugin)]
+#![plugin(rocket_codegen)]
+#![feature(custom_attribute)]
+#![feature(use_extern_macros)]
+
 extern crate clap;
 extern crate csv;
 extern crate env_logger;
 #[macro_use]
 extern crate failure;
-#[macro_use]
+#[macro_use(log, info, debug)]
```

Comment:
I think this should be replaced by just `#[macro_use(log)]` and then all logging macros are called as `log::error!` etc.

Reply:
See https://github.com/SergioBenitez/Rocket/issues/109#issuecomment-300326193 for more info.

Reply:
I attempted this and got compiler errors in `std.rs` 
```text
failed to resolve. Not a module `log`
```
its possible I'm missing something

Reply:
i now see what i was missing